lint-requirements.txt

```diff
@@ -1,2 +1,3 @@
 black==19.10b0
 flake8==3.8.3
+sphinx-panels
```

Comment:
Is `sphinx-panels` a code linter for the documentation?

Reply:
No, it is not. It is used for the dropdown cards. 
Since, there was already a file as `lint-requirements.txt`, that's why I didn't create a new one as `requirements.txt` and added `sphinx-panels` to that file.